site/content/blog/2025-01-09-zap-2-16-0/index.md

```diff
@@ -0,0 +1,111 @@
+and will reduce noise. Some of the standard active scan rules are really targetted at pentesters, and these can waste 
+developers time.
+
+For more details see the new [Scan Policies](/docs/desktop/addons/scan-policies/) add-on.
+
+### Site Tree Export and Pruning
+
+We now have a standard format for representing the Sites Tree in a text file: 
+[Sites Tree Format](/docs/desktop/addons/import-export/sites-tree-format/) .
+This format uniquely identifies nodes in the Sites tree.
+While this might not sound particularly exciting, it does actually allow you to use ZAP to perform "differential" scans,
+i.e. only attacking endpoints that have changed. This means that ZAP can be used to perform really fast testing in CI/CD.
+
+### Sequence Scanning Updates
+
+The [Sequence Scanner](/docs/desktop/addons/sequence-scanner/) add-on has received
+significant attention. It now runs much more quickly and has full Automation Framework support.
+As a result it has been promoted to Beta status.
+
+### Passive Scanner Moved into an Add-On
+
+The passive scanner has now been moved into an add-on: [Passive Scanner](/docs/desktop/addons/passive-scanner/).
+We have an ongoing plan to move as much functionality out of the core as possible, as this allows us to push out fixes and
+enhancements much more quickly.
+
+### Automation Framework Enhancements
+
+The [Automation Framework](/docs/automate/automation-framework/) has a set of new jobs:
+
+* [activeScan-config](/docs/desktop/addons/automation-framework/job-ascanconfig/) This job configures the active scanner, for custom active scans (e.g. Sequence).
+* [activeScan-policy](/docs/desktop/addons/automation-framework/job-ascanpolicy/) This job defines an active scan policy.
+* [export](/docs/desktop/addons/import-export/automation/) This job allows you to export messages in HAR format or as URLs as well as exporting the Sites Tree.
+* [prune](/docs/desktop/addons/import-export/automation/) This job allows you to remove nodes from the Sites Tree using data from a file in the Sites Tree format.
+* [spiderClient](/docs/desktop/addons/client-side-integration/automation/) This job allows you to run the client spider.
+* [exitStatus](/docs/desktop/addons/automation-framework/job-exitstatus/) This job sets ZAP's exit code based on scan results.
+* [sequence-import](/docs/desktop/addons/sequence-scanner/automation/) This job allows you to create a sequence from an HAR file.
+* [sequence-activeScan](/docs/desktop/addons/sequence-scanner/automation/) This job allows you to active scan sequences.
+
+### Checkmarx Rebranding
+
+This release is the first full release with the "ZAP by Checkmarx" branding.
+
+### Docker Hub Organisation Deprecation
+
+We will not use the [softwaresecurityproject](https://hub.docker.com/u/softwaresecurityproject) org for 2.16.0 or following releases.
+
+If you still use this org then please switch to [zaproxy](https://hub.docker.com/u/zaproxy).
```

Comment:
Could also mention GHCR.

Reply:
Yeah, but we get better stats from docker :) 
I'm ok to change this, but the release notes will need to be changed as well ..